tools/packaging/static-build/virtiofsd/build-static-virtiofsd.sh

```diff
@@ -16,10 +16,13 @@ script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 source "${script_dir}/../../scripts/lib.sh"
 
+virtiofsd_repo="${virtiofsd_repo:-}"
 virtiofsd_version="${virtiofsd_version:-}"
+virtiofsd_zip="${virtiofsd_zip:-}"
 
-[ -n "$virtiofsd_version" ] || virtiofsd_version=$(get_from_kata_deps "externals.virtiofsd.version")
+[ -n "$virtiofsd_repo" ] || die "failed to get virtiofsd repo"
 [ -n "$virtiofsd_version" ] || die "failed to get virtiofsd version"
```

Comment:
minor nit and free feel to ignore it and it wasn't introduced in this PR, but is it worth considering using parameter expansion on `$virtiofsd_repo` and `$virtiofsd_version `? I got burned by this in the past so try and be very liberal with using braces everywhere now!